weave-js/src/components/Panel2/PanelTable.styles.ts

```diff
@@ -99,18 +98,23 @@ FilterIcon.displayName = 'S.FilterIcon';
 
 export const ColumnAction = styled.div<{isHovered?: boolean}>`
   cursor: pointer;
+  height: 100%;
+  background-color: transparent;
+`;
+ColumnAction.displayName = 'S.ColumnAction';
+
+export const ColumnActionContainer = styled.div<{isHovered?: boolean}>`
+  display: flex;
   padding: 5px 0px 0px 0px;
+  font-size: 20px;
   flex: 0 0 auto;
   height: 100%;
-  background: white;
```

Comment:
This background was probably here because of sticky headers, have you tested that use-case?

Reply:
I cannot scroll on the table because of the pagination, but I wanted to note that before I changed the divs up to `ColumnActionContainer` (for containing `ColumnAction`s) and `ColumnAction` (for each icon) the CSS for ColumnAction had both `background: white` and `background-color: transparent` in that order.